app/_layouts/extension.html

```diff
@@ -228,7 +228,8 @@
 
       {% if page.type == "plugin" %}
         <hr>
-        <h2 id="terminology">Terminology</h2>
+        <h2 id="terminology">
+        <a class="header-link" href="#terminology" title="Permalink"><span class="sr-only">Permalink</span><i class="fa fa-link"></i></a>Terminology</h2>
```

Comment:
What problem is being solved by this change? Thanks!

Reply:
It is adding the class for the permalink. The href in the anchor tag is adding it to the url (cause now we have an id for our h2 header) that we can now link to from other pages. In this case, now people can link to the terminology section on any HUB page!
Basically, its adding a permalink to the section header